### ISSUE TITLE
Rigid body cache does not bake if a FLIP Fluids domain is in the scene (Blender Bug)

### ISSUE THREAD

**Issue by rlguy** [rlguy/Blender-FLIP-Fluids#409]
**Blender Version (including hash):** Blender 2.80 (e5e885d0ecb9)
**FLIP Fluids Version:** 9.0.5.2 (Experimental)
**Operating System:** Windows 10
**CPU:** Intel Core i7-7700 @ 3.60GHz
**GFX:** GTX 1070 8GB
**RAM:** 32GB

It seems that a Blender rigid body simulation cache is unable to to be baked if there is a FLIP Fluids domain in the scene. Blender only bakes a few frames in the cache and then stops. There are no errors in the system console. After removing the domain, the rigid body simulation bakes without problems.

I suspect that this is a bug in Blender 2.80 since this issue does not affect Blender 2.79. I will need to investigate this further and find out what part of the FLIP Fluids addon scripts is preventing the rigid body simulation from baking so that a bug can be reported to Blender.

Until this issue is fixed, a workaround is to disable the FLIP Fluids domain before baking a rigid body simulation.

Attached is a .blend file that reproduces the issue: 
[issue409.zip](https://github.com/rlguy/Blender-FLIP-Fluids/files/2715956/issue409.zip)

**Comment by rlguy** [comment]
### Update

 It looks like this is a bug in Blender. I have reported the issue here: https://developer.blender.org/T60136

Here is a simplified script which reproduces the issue:

```python
import bpy

def frame_change_pre(scene):    
    new_mesh_data = bpy.data.meshes.new("mesh_name")
    bpy.data.meshes.remove(new_mesh_data)
    
bpy.app.handlers.frame_change_pre.append(frame_change_pre)
```

**Comment by rlguy** [comment]
The bug reported to the developers has been fixed. I have tested the fix and it looks like baking an rigid body cache alongside a FLIP Fluids simulation is still not possible due to another issue: #403. This issue will cause crashes when trying to bake a rigid body cache.

I have added a note to the developers about this here: https://developer.blender.org/T60094#700781

I will close this issue and add this info to #403
